File: tugas/soalB/php/TimBola.php
<?php

    class TimBola{
        private $nama = "",
                $asalNegara = "",
                $sejak = 0,
                $pemain = "";

        // constructor
        public function __construct($name = "", $country = "", $since = 0, $player = ""){
            $this->nama = $name;
            $this->asalNegara = $country;
            $this->sejak = $since;
            $this->pemain = $player;
        }

        // setter
        public function setNama($name){
            $this->nama = $name;
        }
        public function setAsalNegara($country){
            $this->asalNegara = $country;
        }
        public function setSejak($since){
            $this->sejak = $since;
        }
        public function setPemain($player){
            $this->pemain = $player;
        }
        
        // getter
        public function getNama(){
            return $this->nama;
        }
        public function getAsalNegara(){
            return $this->asalNegara;
        }
        public function getSejak(){
            return $this->sejak;
        }
        public function getPemain(){
            return $this->pemain;
        }
    }
    
?>

File: tugas/soalB/python/TimBola.py
class TimBola():
    # atribut
    __nama = "blank"
    __asalNegara = "blank"
    __sejak = 0
    __pemain = "blank"

    # constructor
    def __init__(self, name = "blank", country = "blank", since = 0, player = "blank"):
        self.__nama = name
        self.__asalNegara = country
        self.__sejak = since
        self.__pemain = player

    # setter
    def setNama(self, name):
        self.__nama = name
    def setAsalNegara(self, country):
        self.__asalNegara = country
    def setSejak(self, since):
        self.__sejak = since
    def setPemain(self, player):
        self.__pemain = player
        
    # getter
    def getNama(self):
        return self.__nama
    def getAsalNegara(self):
        return self.__asalNegara
    def getSejak(self):
        return self.__sejak
    def getPemain(self):
        return self.__pemain